TRANSFORMERS/VECTOR_MANIPULATION/VECTOR_INDEXING/example.md

```diff
@@ -0,0 +1 @@
+In this example, we generate a vector by using a `LINSPACE` node. Then, the value at index 100 is extracted at using `VECTOR_INDEXING`. The value is visualized with `BIG_NUMBER`.
```

Comment:
Would be good to add a description here, as well as in the example app using the text feature indicating that we are 0-indexed.